src/frameserver/PointGreyCam.cpp

```diff
@@ -614,12 +617,17 @@ void PointGreyCam<T>::setupGrabSettings()
 template <typename T>
 void PointGreyCam<T>::startCapture()
 {
+    std::cout << "Starting capture...\n";
     // Camera is ready, start capturing images
     pg::Error error = camera_.StartCapture();
     if (error == pg::PGRERROR_ISOCH_BANDWIDTH_EXCEEDED)
-        throw (rte("Interface bandwidth exceeded. Cannot start camera_..\n"));
+      { 
```

Comment:
Try to make sure indentation is correct -- helps with readability and fast review. For anything other than function definitions, curly bracket goes on same line as statement. e.g.

```
if (condition) {
    foo()
} else {
    bar()
}